radio/util/translate.py

```diff
@@ -18,7 +18,9 @@
                        ('\\207', u'š'),
                        ('\\207', u'Š'),
                        ('\\210', u'ú'),
+                       ('\\210', u'Ú'),
                        ('\\211', u'ů'),
+                       ('\\211', u'Ů'),
```

Comment:
These are not in the font!

Reply:
Right, but for monoscreens they are translated to lower case characters now. See the same code for both "ú" and "Ú"

Reply:
Ah yes, OK...